api.md

```diff
@@ -840,7 +842,127 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
 ---
+## Continuous Aggregates :community_function: [](continuous-aggregates)
+TimescaleDB allows users the ability to automatically recompute aggregates 
+at predefined intervals and materialize the results. This is suitable for 
+frequently used queries.
+
+## CREATE VIEW 
+`CREATE VIEW` statement is used to create continuous aggregates.
+
+The syntax is:
+``` sql
+    CREATE VIEW <view_name> [ (column_name [, ...] ) ] 
+    WITH ( timescaledb.<option> = <value> )
+    AS
+    <select-query>
+    
+    <select_query> is of the form :
+    SELECT <grouping-cols>, <aggregate-functions>
+    FROM <hypertable>
+    [WHERE ... ]
+    GROUP BY <time_bucket( <const-value>, partition-col-of-hypertable>),
+             [ optional grouping exprs>
+    [HAVING ...]
+```
+
+#### Parameters
+|Name|Description|
+|---|---|
+| `<view_name>` | Name (optionally schema-qualified) of continuous aggregate view to be created.|
+| `<column_name>`| Optional list of names to be used for columns of the view. If not given, the column names are deduced from the query.|
+ | `WITH` clause | This clause specifies [options](#create-view-with) for the continuous aggregate view.|
+ | `<select_query>`| A `SELECT` query that uses the specified syntax. |
+
+#### `WITH` clause options [](create-view-with)
+|Name|Description|
+|---|---|
+|**Required**||
+ `timescaledb.continuous`|If timescaledb.continuous is not specified, then this is a regular postgres view. |
+|**Optional**||
+|`timescaledb.refresh_lag`|Refresh lag controls the amount by which the materialization will lag behind a the maximum current time value. The default value is twice the bucket width (as specified by the `time_bucket` expression).|
+|`timescaledb.refresh_interval`|Refresh interval controls how often the background materializer is run. It is specified using an `INTERVAL` value. By default, this is set to twice the bucket width (if the datatype is an INTERVAL), otherwise it is set to 12 hours.|
```

Comment:
This should probably be a table, with the type separated out rather than embedded in the description, another column should probably be whether it can be `ALTER`ed. It also looks like we're missing the create_group_indexes param? 

Perhaps this table should appear below the `ALTER` view statement and just be linked to here?

Reply:
oh. and potentially a separate column for default and description?

Reply:
added.